components/expand-collapse/expand-collapse-content.js

```diff
@@ -47,8 +47,8 @@ class ExpandCollapseContent extends LitElement {
 			}
 
 			/* prevent margin colapse on slotted children */
-			.d2l-expand-collapse-content-inner:before,
-			.d2l-expand-collapse-content-inner:after {
+			.d2l-expand-collapse-content-inner::before,
```

Comment:
Found a bunch of these cases where we were incorrectly using pseudo-class syntax (single `:`) instead of pseudo-element (doubel `::`).